standalone/WebSocketExportToOrg.py

```diff
@@ -0,0 +1,141 @@
+"""
+This script export WebSocket communication to Emacs Org File.
+Script require the external Module PyOrgMode (https://github.com/bjonnh/PyOrgMode)
+To add an external module: Tools -> Options -> Jython 
+
+Export Format:
+
+* (#<ChannelID>)<WebSocket Channel host name>
+** Summary [Optional]
+	 - Host: <host>
+	 - URI: <URI>
+	 - Port: <port>
+	 - Websocket Port: <WebSocket Port>
+	 - Websocket URL: <WebSocket URL>
+	 - Messages Opcodes: 
+	 - Outgoing Messages format: 
+	 - Incoming Messages format: 
+	 - Input Messages encryption: 
+	 - Output Messages encryption: 
+	 - Ping/Pong: 
+	 - Close message: Opcode= , Text= 
+** Switching Protocol
+*** Handshake Request
+	#+BEGIN_SRC html
+	<Http Handshake Request>
+	#+END_SRC
+*** Handshake Respone
+	#+BEGIN_SRC html
+	<Http Handshake Request>
+	#+END_SRC
+** Messages
+*** (#<ChannelID>.<MessageID) <Direction> <Opcode> <Date Time> <Byte Lenght>
+	- Text
+	<Text Payload>
+	- Hex
+	<Hex Payload>
+"""
+
+from org.parosproxy.paros.control import Control
+from org.parosproxy.paros.extension.history import ExtensionHistory
+from org.zaproxy.zap.extension.websocket import ExtensionWebSocket
+from org.zaproxy.zap.extension.websocket import WebSocketChannelDTO
+from org.zaproxy.zap.extension.websocket import WebSocketMessageDTO
+
+import PyOrgMode
+import sys
+import binascii
+
+reload(sys)
+sys.setdefaultencoding('utf-8')
+
+extWebSocket = Control.getSingleton().getExtensionLoader().getExtension(ExtensionWebSocket.NAME)
+extHist = Control.getSingleton().getExtensionLoader().getExtension(ExtensionHistory.NAME)
+pathToOrgFile = "/path/to/file.org"
+printSummary = True
+base = PyOrgMode.OrgDataStructure()
+channelsList = extWebSocket.getStorage().getTable().getChannelItems()
+
+for channel in channelsList:
+  handshakeMessage = channel.getHandshakeReference().getHttpMessage()
```

Comment:
Note that the `handshakeMessage` will be null if the message was deleted (not usual but can happen).